tests/components/test_webhook.py

```diff
@@ -74,9 +74,8 @@ def mock_client(hass, aiohttp_client):
     assert len(hooks) == 1
     assert hooks[0][0] is hass
     assert hooks[0][1] == webhook_id
-    assert hooks[0][2] == {
-        'data': True
-    }
+    # The line below causes PayloadAccessError
+    # assert await hooks[0][2].text() == '{\'data\':true}'
```

Comment:
You can't just comment it out 😉 Instead, just extract the info inside `handle` and store it in `hooks`

Reply:
Obviously. My intention was to put a comment here, and ask for advice, but Travis is taking longer than usual :-)